hudi-io/src/main/java/org/apache/hudi/io/hfile/HFileReaderImpl.java

```diff
@@ -281,7 +282,7 @@ Map<Key, BlockIndexEntry> getDataBlockIndexMap() {
    * @throws IOException upon error.
    */
   private static HFileTrailer readTrailer(SeekableDataInputStream stream,
-                                          long fileSize) throws IOException {
+      long fileSize) throws IOException {
```

Comment:
nit: indentation does not follow the checkstyle

Reply:
Will revert.